src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx

```diff
@@ -122,7 +126,7 @@ export default function TokenTabContent({ onFinish }: TabContentProps) {
 
   return (
     <Row gutter={32}>
-      <Col md={10} xs={24}>
+      <Col md={!hidePreview ? 10 : 24} xs={24}>
```

Comment:
we should make a layout component for this stuff eventually to reduce duplication

Reply:
Yeah true that.